databases/week2/part3.sql

```diff
@@ -0,0 +1,35 @@
+-- Part 3: More queries
```

Comment:
you should use lesson2 data for this part, 
https://github.com/HackYourFuture-CPH/databases/blob/master/lesson2/lesson2-data.sql

Reply:
I did use this data, but as we used it previously, it was already saved.